CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Breaking changes
```

Comment:
@clayton-cornell Should we find a different way of classifying these kinds of changes for non-stable functionality now that we're post-1.0?

Reply:
maybe append to this header? `Breaking changes (Public preview features)`

Reply:
cc @richardqlam on this too

Reply:
Hm I'm not sure further sub-categorizing breaking changes at a top level is all that useful. Feels like a changelog is most consumable when it talks about things on a feature-by-feature level. And then within the context of each feature highlighting what state it is (GA, public preview, etc).

Reply:
Just looking at this now. It's still a breaking change, even though it's in a public preview state. The info clearly says it's an update to the public preview.... to my way of thinking, it's clear enough as-is.